content/blog/pulumi-service-improvements_02-2020/index.md

```diff
@@ -0,0 +1,104 @@
+---
+title: "Pulumi Console Improvements, February 2020"
+h1: "Pulumi Console Improvements, February 2020"
+authors: ["chris-smith"]
+tags: ["pulumi","new-features"]
+date: "2020-02-28"
+meta_desc: "Recent improvements to the Pulumi Console: stack tags, audit logs, CI/CD integration, downloadable checkpoint files"
+meta_image: "pretty-print-multiline-json.gif"
+---
+
+Hello! Many posts here on the [Pulumi blog]({{< relref "/blog" >}}) describe new features in our core libraries or
```

Comment:
I think we can likely just skip this paragraph - lead in with what's exciting instead of what sounds a little like an excuse.

Reply:
Fair enough, done.